oqs/src/macros.rs

```diff
@@ -10,7 +10,7 @@ macro_rules! newtype_buffer {
         #[derive(Debug, Clone, PartialEq, Eq)]
         #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
         pub struct $name {
-            bytes: Vec<u8>,
+            pub bytes: Vec<u8>,
```

Comment:
I will not merge this: it does not provide faster reference than `as_ref` and this allows to mutually access `bytes` which I do not think is a) idiomatic Rust and b) good and safe API design.

Reply:
We need the ability to construct this given a `Vec<u8>`. The `AsRef<u8>` impl is only good for reading the data.

Reply:
So, if you're interesed in idiomatic rust, we could do:

```rust
impl From<Vec<u8>> for $name {
    fn from(bytes: Vec<u8>>) -> $name {
        $name { bytes }
    }
}